File: small_problems/easy_5/ex1.rb

def ascii_value(sentence)
  return 0 if sentence == ''
  sentence.chars.reduce(0) do |memo, char|
    memo += char.ord
  end
end

p ascii_value('Four score') == 984
p ascii_value('Launch School') == 1251
p ascii_value('a') == 97
p ascii_value('') == 0

File: js_small_problems/string_processing/ex1.js
function isUppercase(string) {
  string = string.replace(/[^a-z]/gi, '');
  console.log(string.toUpperCase() === string);
}

isUppercase('t');               // false
isUppercase('T');               // true
isUppercase('Four Score');      // false
isUppercase('FOUR SCORE');      // true
isUppercase('4SCORE!');         // true
isUppercase('');                // true